### DISCUSSION TITLE
terminal app

### DISCUSSION THREAD

**Discussion by hitesh7424** [vedicscript/vedicscript#12] [Q&A] [answered]
Which terminal app shows Hindi correctly?
thanks
love from me 💟

**Comment by PtPrashantTripathi** [comment, marked as answer]
That's a good question basically if you use a Linux machine then use  Konsole it support devnagari better then default gnome terminal and but if use windows or Mac it's frustrating there i don't know why but Microsoft cmd support emojis which are basically not a charcter. But it doesn't support devnagari charcter 😐 it just displays dotted o something idk why.But here is one alternate terminal for Mac and windows which you can use https://github.com/Eugeny/tabby/releases/latest it's suppourt devnagari.On 23-Mar-2023 12:48 am, hitesh7424 ***@***.***> wrote:
Which terminal app shows Hindi correctly?
thanks
love from me 💟

**Reply by hitesh7424** [reply]
Thanks for response
I guess devnagari is not for windows terminal

![image](https://user-images.githubusercontent.com/75922163/227249608-3f5c50c4-d562-4605-a206-76295b192541.png)
